### PR TITLE
Updated README.md with details

### PR DESCRIPTION
**Pull request by VikasKumarC**
Signed-off-by: Vikas Kumar Choudhary <vikas.kumar@seagate.com>

# Problem Statement
-  Problem statement

# Design
-  For Bug, Describe the fix here.
-  For new test addition post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is formatted (e.g. using pycharm in-built formatter)

# Testing 
  Checklist for Author
-  [ ] New/Affected tests are executed
-  [ ] Attach test execution logs

# Review Checklist 
  Checklist for Author
-  [ ] JIRA number/GitHub Issue added to PR
-  [ ] PR is self reviewed
-  [ ] Jira and state/status is updated and JIRA is updated with PR link
-  [ ] Check if the description is clear and explained

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe

-----
[View rendered README.md](https://github.com/VikasKumarC/corio/blob/README_Updates/README.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,88 @@
...
+    Use following command to execute workloads from different scenarios in parallel.
+
+    python corio.py -ak <access_key> -sk <secret_key> -ti config/io
+
+    Prerequisite: A S3 account and access key and secret key should be present to carry out execution.
+
+    here test input may be either a single yaml file path i.e. config/s3/s3api/bucket_operations.yaml
+    or
+    A Directory path containing multiple tests data input yaml files those need to be executed in parallel
+    i.e. config/s3/s3api or any other customized directory location may be created containing test data input yaml files and can be passed as argument
```
Comment (ketanarlulkar) [inline README.md]: This can be outside of the block.

File: README.md [hunk 2]
```diff
@@ -1 +1,88 @@
...
+    > origin    https://github.com/YOUR_USERNAME/corio.git (fetch)
+    > origin    https://github.com/YOUR_USERNAME/corio.git (push)
+    > upstream        https://github.com/Seagate/corio.git (fetch)
+    > upstream        https://github.com/Seagate/corio.git (push)
+    git fetch upstream
+    git pull upstream dev
+
+Set up dev environment
+    Following steps helps to set up env, where corio runs. These steps assume that you have installed git client and cloned repo.
+    1. Python Latest Version should be installed and configured in Client System
```
Comment (ketanarlulkar) [inline README.md]: Are we really supporting the latest python version? We need to stick to some fixed version.

File: README.md [hunk 3]
```diff
@@ -1 +1,88 @@
...
+    git remote -v
+    git remote add upstream https://github.com/Seagate/corio.git
+    git remote -v
+    Issuing the above command again will return you output as shown below.
+    > origin    https://github.com/YOUR_USERNAME/corio.git (fetch)
+    > origin    https://github.com/YOUR_USERNAME/corio.git (push)
+    > upstream        https://github.com/Seagate/corio.git (fetch)
+    > upstream        https://github.com/Seagate/corio.git (push)
+    git fetch upstream
+    git pull upstream dev
```
Comment (DhobaleVishal) [inline README.md]: To fetch upstream changes in fork repo we need to perform rebase along with pull. ex: git pull upstream --rebase dev

File: README.md (hunk 2 above)
Comment (ketanarlulkar) [inline README.md]: All these steps are being rendered as single paragraph instead of numbered list. Please correct it.

File: README.md [hunk 4]
```diff
@@ -1 +1,88 @@
...
+    4. Change dir to corio project directory, make sure a requirement file is present in project dir
+    5. Create Virtual environment `python3.7 -m venv virenv`
+    6. Activate Virtual environment `source virenv/bin/activate`
+    7. Use following command to install python packages.
+            `pip install --ignore-installed -r requirements.txt`
+
+Steps to Configure test scripts for execution in parallel mode.
+    Use following command to execute workloads from different scenarios in parallel.
+
+    python corio.py -ak <access_key> -sk <secret_key> -ti config/io
```
Comment (DhobaleVishal) [inline README.md]: endpoint fqdn/url should be passed through command.

File: README.md [hunk 5]
```diff
@@ -1 +1,88 @@
...
+                            help="Directory path containing test data input yaml files or "
+                                "input yaml file path."
+
+    Optional Arguments:
+
+        "-us", "--use_ssl",
+                            type=lambda x: bool(strtobool(str(x))), default=True,
+                            help="Use HTTPS/SSL connection for S3 endpoint."
+        "-sd", "--seed", type=int, help="seed.",
+                            default=random.SystemRandom().randint(1, 9999999)
```
Comment (ketanarlulkar) [inline README.md]: Describe seed argument.

File: README.md [hunk 6]
```diff
@@ -1 +1,88 @@
-This is under development project, Use with Caution.
+# This is under development project, Use with Caution
+
```
Comment (DhobaleVishal) [inline README.md:2]: Is this expected to have a python installation steps mentioned in readme? If tool supports only specific python version then I think so.

**Review by DhobaleVishal — CHANGES_REQUESTED** [review]

**Comment by kulrk** [comment]
should we document about existing test plans and how to add a new one?

File: README.md (hunk 6 above)
Comment (VikasKumarC) [reply README.md:2]: Updated with Python3.7

File: README.md (hunk 2 above)
Comment (VikasKumarC) [reply README.md]: Done changes

File: README.md (hunk 5 above)
Comment (VikasKumarC) [reply README.md]: Done

File: README.md (hunk 4 above)
Comment (VikasKumarC) [reply README.md]: done

**Review by DhobaleVishal — APPROVED** [review]

**Review by ketanarlulkar — APPROVED** [review]

**Pull request merged**